_posts/2020-02-10-newpipe-0.18.3-released.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: "NewPipe 0.18.3 released: Hotfix for the SoundCloud Extractor"
+short: "NewPipe 0.18.3 released"
+date: 2020-02-10 00:00:00 +0100
+categories: [release, pinned]
+author: Stypox
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+A change on SoundCloud side made it necessary to make some edits to the SoundCloud Extractor, so a new update for NewPipe is here!
+
+## Hotfix
```

Comment:
I'm not a fan of including any kind of headlines in excerpts. One shouldn't include half of the article and split off the rest. IMO the text above is sufficient to inform people what the post is about, anyone who wants more details can open the full article.

Reply:
I agree with you, the intermediary title is quite useless, but what I did is just copying the layout of the 0.18.2 article ;-)
Is it ok now?

Reply:
We've had a similar discussion there. Yeah, it's okay now. There's one remaining issue I've found, when it's fixed we can merge and deploy the new page.